backend/src/api/test/entities/test.entity.ts

```diff
@@ -0,0 +1,25 @@
+import {TestInterface} from "../interfaces/test.interface";
+import {Column, Entity, ManyToOne, PrimaryGeneratedColumn} from "typeorm";
+import {OwnerInterface} from "../../owner/interfaces/owner.interface";
+import {OwnerEntity} from "../../owner/entities/owner.entity";
+import {SettingsInterface} from "../../settings/interfaces/settings.interface";
+import {SettingsEntity} from "../../settings/entities/settings.entity";
+
+@Entity("Test")
+export class TestEntity implements TestInterface
+{
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @ManyToOne(() => SettingsEntity, settings => settings.tests, {cascade :true})
+    settings: SettingsInterface;
+
+    @ManyToOne(() => OwnerEntity, owner => owner.tests, {cascade :true})
+    owner : OwnerInterface;
```

Comment:
Do we need to store a `owner`? I thought we'll only send and save `owner_link`

Reply:
Nice catch. But that's taken straight from conceptual diagram. Maybe that needs a redo

Reply:
From this one? 
![image](https://user-images.githubusercontent.com/6399265/148740328-e11ad486-f929-4b2a-a74f-60d9dd5206f7.png)

Reply:
No, I mean this
![image](https://user-images.githubusercontent.com/53041150/148785782-8a9b73be-d20e-49c4-859a-25736acbb4d2.png)